src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -204,8 +204,8 @@ which we divide into several categories:
       "InjectionStart": "REQUIRED",
```

Comment:
Should the added description here be, "Corresponds to DICOM tag (0018,1072) RadioPharamaceuticalStopTime."? I'm wondering if your dicoms just don't have this field. The one I'm looking at now is missing `(0018, 1073)` @CPernet @mnoergaard

Reply:
?? how does Start match StopTime

Reply:
I don't have tag `(0018, 10430` in my dicom set, should this description then include both tags? Here is what I am looking at:

```
MH02144183MACLT:pet_wb_dyn_tof1 galassiae$ dcmdump "ZE56.PT.PET_PET_RH_DOSIMETRY_11C.IMA" | grep "(0018,1042)"
MH02144183MACLT:pet_wb_dyn_tof1 galassiae$ dcmdump "ZE56.PT.PET_PET_RH_DOSIMETRY_11C.IMA" | grep "(0018,1072)"
    (0018,1072) TM [104222.000]                             #  10, 1 RadiopharmaceuticalStartTime
MH02144183MACLT:pet_wb_dyn_tof1 galassiae$
```